src/ConfigManager.cpp

```diff
@@ -505,9 +502,13 @@ void ConfigManager::handleAPPost() {
   }
 
   storeWifiSettings(ssid, password);
+  if (this->getMode() != station) {
+    this->writeConfig();
```

Comment:
Looks good, had a thought that this write _could_ live in the `cold start` handler

Reply:
Just saw your other comment 👍